tools/packaging/packaging.strings.psd1

```diff
@@ -38,6 +48,18 @@ case "$1" in
         exit 0
     ;;
 esac
+
+if [ -f /usr/lib/x86_64-linux-gnu/libssl.so.1.1 ] ; then
```

Comment:
Ditto here, creating a symlink that points to the wrong OpenSSL version library is not the right thing to do here.

Reply:
See https://github.com/PowerShell/PowerShell/pull/15109#discussion_r602769825